gohttp.go

```diff
@@ -138,7 +154,8 @@ func SerializeRequest(r *http.Request) ([]byte, error) {
 //
 // If the user allows LF line endings, the header fields and the empty
```

Comment:
```suggestion
// The option AllowLFLineEndings allows the header fields and the empty
// line terminating the header section to be LF instead of CRLF endings. 
```

Reply:
I would extend the description to something like this